src/graphics/device.js

```diff
@@ -879,21 +879,22 @@ Object.assign(pc, function () {
             this.transformFeedbackBuffer = null;
         },
 
-        initializeGrabPassTexture: function () {
+        initializeGrabPass: function (options) {
             if (this.grabPassTexture) return;
 
+            var format = options.alpha ? pc.PIXELFORMAT_R8_G8_B8_A8 : pc.PIXELFORMAT_R8_G8_B8;
+
             var grabPassTexture = new pc.Texture(this, {
-                format: pc.PIXELFORMAT_R8_G8_B8_A8,
-                autoMipmap: false
+                format: format,
+                minFilter: pc.FILTER_LINEAR,
+                magFilter: pc.FILTER_LINEAR,
+                addressU: pc.ADDRESS_CLAMP_TO_EDGE,
+                addressV: pc.ADDRESS_CLAMP_TO_EDGE,
+                mipmaps: false
```

Comment:
So does the grab pass texture now never have correct ```width``` and ```height``` set?

Reply:
Nop, and we don't actually need to since the texture is not meant to be used by js.